libgraphics/shaders/CMakeRule.txt

```diff
@@ -0,0 +1,24 @@
+######################
+# SHADER COMPILATION #
+######################
+function(add_shader TARGET SHADER)
```

Comment:
Can you add an example showing how to use this function ?

Reply:
```suggestion
# add_shader(${TARGET} shader.(vert|frag|comp))
function(add_shader TARGET SHADER)
```